array.c

```diff
@@ -0,0 +1,39 @@
+//Emma
+//returning an arry with sum
+
+
+#include <stdio.h>
```

Comment:
Nice job on doing this in a super efficient way! The only thing is I'd recommend having the user input the array, but that's not as important. Nice job!